lib/budgie_web/live/budget_show_live.html.heex

```diff
@@ -8,10 +8,19 @@
     module={BudgieWeb.TransactionDialog}
     id="transaction"
     budget={@budget}
-    transaction={assigns[:transaction] || default_transaction()}
+    transaction={assigns[:transaction] || default_transaction(@budget)}
     action={@live_action}
   />
 </.modal>
+
+<div class="mb-6">
```

Comment:
Not shown in video: back navigation to the budget main page. Little quality of life thing.